api/src/main/java/uk/gov/dvsa/mot/trade/api/MotTest.java

```diff
@@ -64,6 +65,16 @@ public void setOdometerUnit(String odometerUnit) {
         this.odometerUnit = odometerUnit;
     }
 
+    public String getOdometerResultType() {
+
+        return odometerResultType;
+    }
+
+    public void setOdometerResultType(String odometerResultType) {
+
+        this.odometerResultType = odometerResultType;
```

Comment:
Your `RfrAndAdvisoryItem.java` class uses a fluent API, why doesn't this one? Consistency is key

Reply:
I try to keep styles of classes consistent; The other 11 accessors and modifiers in this class do not use fluent API.